packages/apputils-extension/src/index.ts

```diff
@@ -340,19 +340,26 @@ async function updateTabTitle(workspace: string, db: IStateDB, name: string) {
       workspace.startsWith('auto-') ? ` (${workspace})` : ``
     }`;
   } else {
-    // First 15 characters of current document name
-    current = current.split(':')[1].slice(0, 15);
+    // File name from current path
+    let currentFile: string = PathExt.basename(current)?.split(':')[0];
```

Comment:
```suggestion
    let currentFile: string = PathExt.basename(current.split(':')[1]);
```

Reply:
My thinking is that if something which is not a proper path gets passed to `PathExt.basename` its behaviour may be unexpected, so let's extract the path early and only then pass it to PathExt. The details may need to be adjusted.

Reply:
Checked this and commited.